pkg/sessions/redis/redis_store.go

```diff
@@ -32,9 +34,10 @@ type TicketData struct {
 // SessionStore is an implementation of the sessions.SessionStore
 // interface that stores sessions in redis
 type SessionStore struct {
-	CookieCipher  *encryption.Cipher
-	CookieOptions *options.CookieOptions
-	Client        Client
+	SessionOptions *options.SessionOptions
+	CookieCipher   *encryption.Cipher
```

Comment:
Is the CookieCipher not already a member of CookieOptions?

Reply:
Good point, removed the redundancy